src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -10,14 +10,16 @@
 #include "ConptyConnection.h"
 
 #include <windows.h>
+//#include <winrt/Windows.Foundation.Collections.h>
```

Comment:
Remove dead code (and below)

Reply:
Will do. I commented and uncommented this line so many times trying to kill the squiggles of death that I lost track of it.